### PR TITLE
vifm: new homepage, add HEAD

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/vifm.rb [hunk 1]
```diff
@@ -12,10 +17,17 @@ class Vifm < Formula
...
+    else
+      system "./configure", *args
+    end
     ENV.deparallelize
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    pipe_output("script -q /dev/null #{bin}/vifm", ":q")
```
Comment (bfontaine) [inline Library/Formula/vifm.rb]: Please add `, 0` at the end to ensure the return code is `0`:

```
pipe_output("script -q /dev/null #{bin}/vifm", ":q", 0)
```

**Comment by dunn** [comment]
Ack, I'm out of ideas about how to get a passing test (a passing test that isn't a false positive).

**Comment by bfontaine** [comment]
> Error opening terminal: unknown.

You could try setting `ENV["TERM"]`.

**Comment by dunn** [comment]
Thanks, @bfontaine!

**Comment by bfontaine** [comment]
Thank *you*, @dunn!

**Pull request closed without merging**